projects/ad9695_fmc/Readme.md

```diff
@@ -0,0 +1,10 @@
+
+# AD9695-EVAL HDL reference design
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/eval-ad9695.html)
+  * Parts : [14-Bit, 1300 MSPS/625 MSPS, JESD204B, Dual Analog-to-Digital Converter](https://www.analog.com/ad9695.html)
+  * Project Doc: https://wiki.analog.com/resources/eval/ad9695-1300ebz
```

Comment:
The project DOC should point to the same link as the HDL doc, as that's the link for the reference design documentation